Android.mk

```diff
@@ -9,8 +9,6 @@ LOCAL_SRC_FILES := \
 LOCAL_MODULE := dirtycow
 LOCAL_LDFLAGS   += -llog
 LOCAL_CFLAGS    += -DDEBUG
-LOCAL_CFLAGS    += -fPIE
```

Comment:
Unfortunately `-fPIE` is needed for 5 (I think) and up. Can we ifdef it somehow?